### ISSUE TITLE
FutureWarnings from geopandas 0.6.0

### ISSUE THREAD

**Issue by QuLogic** [ResidentMario/geoplot#200]
Due to geopandas/geopandas#1085 and geopandas/geopandas#1062, there are several of these `FutureWarning`s in the tests:
```
tests/mixin_tests.py::TestPlot::test_init_extent_axes
tests/mixin_tests.py::TestPlot::test_init_extent_geoaxes
tests/viz_tests.py::test_clip_params_overlay[kwargs0]
tests/viz_tests.py::test_clip_params_overlay[kwargs1]
tests/viz_tests.py::test_param_extent_unproj
tests/viz_tests.py::test_param_extent_proj
  /builddir/build/BUILDROOT/python-geoplot-0.4.0-1.fc31.x86_64/usr/lib/python3.7/site-packages/geoplot/geoplot.py:625: FutureWarning:     You are passing non-geometry data to the GeoSeries constructor. Currently,
      it falls back to returning a pandas Series. But in the future, we will start
      to raise a TypeError instead.
    extent = gpd.GeoSeries(self.extent) if self.extent is not None else None

tests/mixin_tests.py::TestHue::test_hue_init_hue
  /builddir/build/BUILD/geoplot-0.4.0/tests/mixin_tests.py:205: FutureWarning:     You are passing non-geometry data to the GeoSeries constructor. Currently,
      it falls back to returning a pandas Series. But in the future, we will start
      to raise a TypeError instead.
    hue = gpd.GeoSeries(np.random.random(100))

tests/mixin_tests.py::TestHue::test_hue_init_hue
tests/mixin_tests.py::TestScale::test_scale_init_scale_dtypes
tests/viz_tests.py::test_hue_params[kwargs9]
tests/viz_tests.py::test_hue_params[kwargs10]
  /builddir/build/BUILDROOT/python-geoplot-0.4.0-1.fc31.x86_64/usr/lib/python3.7/site-packages/geoplot/geoplot.py:1797: FutureWarning:     You are passing non-geometry data to the GeoSeries constructor. Currently,
      it falls back to returning a pandas Series. But in the future, we will start
      to raise a TypeError instead.
    return gpd.GeoSeries(var, index=df.index)

tests/mixin_tests.py::TestScale::test_scale_init_scale_dtypes
  /builddir/build/BUILD/geoplot-0.4.0/tests/mixin_tests.py:376: FutureWarning:     You are passing non-geometry data to the GeoSeries constructor. Currently,
      it falls back to returning a pandas Series. But in the future, we will start
      to raise a TypeError instead.
    scale = gpd.GeoSeries(np.random.random(100))

tests/mixin_tests.py::TestScale::test_scale_init_scale_dtypes
  /builddir/build/BUILD/geoplot-0.4.0/tests/mixin_tests.py:383: FutureWarning:     You are passing non-geometry data to the GeoSeries constructor. Currently,
      it falls back to returning a pandas Series. But in the future, we will start
      to raise a TypeError instead.
    scale = gpd.GeoSeries(np.random.random(100))

tests/viz_tests.py::test_clip_params_geometric[kwargs0]
tests/viz_tests.py::test_clip_params_geometric[kwargs2]
  /usr/lib/python3.7/site-packages/geopandas/geoseries.py:290: UserWarning: GeoSeries.isna() previously returned True for both missing (None) and empty geometries. Now, it only returns True for missing values. Since the calling GeoSeries contains empty geometries, the result has changed compared to previous versions of GeoPandas.
  Given a GeoSeries 's', you can use 's.is_empty | s.isna()' to get back the old behaviour.
  
  To further ignore this warning, you can do: 
  import warnings; warnings.filterwarnings('ignore', 'GeoSeries.isna', UserWarning)
    return self.isna()
```

**Comment by QuLogic** [comment]
This is like #187, which was fixed in #188, but only in one place.

**Comment by ResidentMario** [comment]
Fixed in #219.
